open-source.adoc

```diff
@@ -206,83 +206,81 @@ speaking, you should donate to me based on the utility you find my
 permissionless nature.
 
 Arguably, the most well-known pseudonym in Bitcoin is Satoshi
-Nakamoto. It's unclear what his reasons for using a pseudonym was, but
-in hindsight, it was probably a good decision for multiple reasons:
+Nakamoto. It's unclear why he chose to do so, but
+with hindsight it was probably a good decision for multiple reasons:
 
-* As many people speculate that Nakamoto has a lot of bitcoin, it's
+* As many people speculate that Nakamoto owns a lot of bitcoin, it's
 imperative for his financial and personal safety to keep his identity
 unknown.
-* Since his identity is unknown, there is no person to prosecute,
-  giving various government authorities a hard time.
+* Since his identity is unknown, there is no possibility of persecuting anyone, which
+  gives various government authorities a hard time.
 * There is no authoritative person to look up to, making Bitcoin more
   meritocratic and resilient against blackmailing.
 
-Notice how these points don't just hold for Satoshi Nakamoto, but for
-anyone who works in Bitcoin or holds significant amounts of the
+Notice that these points don't just hold true for Satoshi Nakamoto, but for
+anyone working in Bitcoin or holding significant amounts of the
 currency, to varying degrees.
 
 [[selectioncryptography]]
 === Selection cryptography
 
-Open source developers often use open source libraries developed by
-other people. This is a natural, and awesome part of a healthy
-ecosystem. But Bitcoin software deals with real money, and as such,
-developers need to be extra careful when choosing which third party
-libraries the software should depend on.
-
-In a
-https://btctranscripts.com/greg-maxwell/2015-04-29-gmaxwell-bitcoin-selection-cryptography/[philosophical
-talk about cryptography] (search for the part that starts with "`The
-art of selection cryptography`" or watch the
-https://youtu.be/Gs9lJTRZCDc?t=2236[video]), Gregory Maxwell wants to
-redefine the term cryptography which he believes is too narrow. He
-speaks about how, fundamentally, _information wants to be free_, and
+Open source developers often make use of open source libraries developed by
+other people. This is a natural and awesome part of any healthy
+ecosystem. But Bitcoin software deals with real money, and as such it needs to be extra careful when choosing which third party libraries it should depend on.
+
+In a philosophical
+https://btctranscripts.com/greg-maxwell/2015-04-29-gmaxwell-bitcoin-selection-cryptography/[talk
+about cryptography] (you may find the video
+https://youtu.be/Gs9lJTRZCDc?t=2236[here]), Gregory Maxwell wants to
+redefine the term "`cryptography`" which he believes to be too narrow. He
+explains that fundamentally _information wants to be free_, and
 makes his definition of cryptography based on that:
 
-[quote, Gregory Maxwell, The Art Of Selection Cryptography]
+[quote, Gregory Maxwell, Bitcoin Selection Cryptography (2015)]
```

Comment:
I'm not sure about the title of this talk.

The transcript says "Bitcoin Selection Cryptography"
The first slide says "You paid a bitcent, here's my thoughts" 
The first slide of the section of the talk that's about selection cryptography is "The art of Selection Cryptography": https://youtu.be/Gs9lJTRZCDc?t=2236

Reply:
Never mind. Bitcoin Selection Cryptography is probably the best option here.